server/src/routers/converterRouter.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+const router = new express.Router();
+
+const { mapping } = require('../constants/config');
+const { parseInputFile, sendResponse } = require('../util');
+
+router.use(express.json());
+
+var Converter = require('api-spec-converter');
+
+//Endpoint for conversion
+router.post('/api/specification/:format/:convertTo', upload.single('file'), (req, res) => {
+    const {format, convertTo} = req.params,
+        {targetSyntax = 'json', toFile = false} = req.query;
+
+    // Check if the conversion is possible
+    if(!(mapping[format] && mapping[format].includes(convertTo))) {
+      return res.status(404).send({'message': 'Conversion mapping not found.'});
+    }
+    else {
+      const file = req.file;
+
+      if(!file) {
+        return res.status(400).send({'message': 'File not available.'});
+      }
+
+      //Parse JSON / YAML Input String
+      const parsedSpec = parseInputFile(file);
```

Comment:
if this bit is so complex that you need a comment to explain it, extract it into a method that says parseInputString(). Remove the comment, ask yourself if it's helping in a way that your code can't @pcgamer1